### DISCUSSION TITLE
fatal: not a git repository (or any of the parent directories): .git

### DISCUSSION THREAD

**Discussion by lexiroro** [community/community#31691] [Other Feature Feedback, Questions, & Ideas]
Lexis-MacBook-Air-2:~ lexi_roro$ cd Downloads
Lexis-MacBook-Air-2:Downloads lexi_roro$ mkdir my-second-repo
Lexis-MacBook-Air-2:Downloads lexi_roro$ cd my-second-repo
Lexis-MacBook-Air-2:my-second-repo lexi_roro$ git init
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Initialized empty Git repository in /Users/lexi_roro/Downloads/my-second-repo/.git/
Lexis-MacBook-Air-2:my-second-repo lexi_roro$ git remote
Lexis-MacBook-Air-2:my-second-repo lexi_roro$ cd ..
Lexis-MacBook-Air-2:Downloads lexi_roro$ cd my-first-repo
Lexis-MacBook-Air-2:my-first-repo lexi_roro$ git remote -v
fatal: not a git repository (or any of the parent directories): .git

Not sure why I am getting the error message. In my class, it says I should be expecting after "git remote -v": 
origin git @github.com: GitTutorials101/my-first-repo.git (fetch)
origin git @github.com: GitTutorials101/my-first-repo.git (push)

**Comment by toufiqul-hasan** [comment]
Read this:
https://www.datree.io/resources/git-error-fatal-not-a-git-repository

**Comment by airtower-luna** [comment]
You did `git init` in the "second repo" directory here:

```
 Lexis-MacBook-Air-2:my-second-repo lexi_roro$ git init
```

But never in the first one, as far as the quoted part shows. That's where the "not a git repository" message comes from.

```
Not sure why I am getting the error message. In my class, it says I should be expecting after "git remote -v":
origin git @github.com: GitTutorials101/my-first-repo.git (fetch)
origin git @github.com: GitTutorials101/my-first-repo.git (push)
```

`git init` creates a new repository without any content or remotes. `git clone` will create a copy of a remote repository *and* set it up as the `origin` remote. You can always add a remote manually, if you want.

See: [Git Basics - Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)
